tensorflow/contrib/rccl/python/ops/rccl_ops.py

```diff
@@ -270,8 +270,9 @@ def _maybe_load_rccl_ops_so():
   with _module_lock:
     global _rccl_ops_so
     if not _rccl_ops_so:
-      _rccl_ops_so = loader.load_op_library(
-          resource_loader.get_path_to_datafile('_rccl_ops.so'))
+      library_filename = os.path.join(resource_loader.get_data_files_path(),
+                                    '_rccl_ops.so')
+      _rccl_ops_so = load_library.load_op_library(library_filename)
```

Comment:
@sumesh13  could you clarify if changes in this file really needed to get XLA working? they don't seem to be relevant for the purpose of this PR

Reply:
No, likely rebase will help